fixtures/_integration.cpp.sem.uast

```diff
@@ -87,31 +88,8 @@
                      IsActive: true,
                      IsFrozen: true,
                      IsLValue: false,
-                     Operator: "left <<",
-                     'Prop_InitOperand2': { '@type': "CPPASTLiteralExpression",
```

Comment:
The node disappeared. Comment in the native driver tells that it's okay, but I wonder what kind of C++ construct can add an initializer for the second operand in the binary expression. I would suggest to leave the native AST as-is with duplicates and hide them during transform if they are really the same. It can be done by marking both fields with the same variable name. In case nodes will be different, the SDK will fail the transform and we will know what case can generate this AST.

Reply:
Good idea.

Reply:
PS: Done. Doesn't seem to detect any non-duplicated with the current fixtures, but I'll leave it so we can detect in the future and check the shape the code takes for a non dup to appear.